src/MongoDB.Driver.Core/Core/Operations/ChangeStreamOperation.cs

```diff
@@ -42,6 +42,14 @@ public interface IChangeStreamOperation<TResult> : IReadOperation<IAsyncCursor<T
         /// </value>
         BsonDocument ResumeAfter { get; set; }
 
+        /// <summary>
+        /// Gets or sets the start after.
+        /// </summary>
+        /// <value>
+        /// The start after.
+        /// </value>>
+        BsonDocument StartAfter { get; set; }
+
```

Comment:
Add a new property for the cursor to use to record the most recently seen document result token:

    BsonDocument DocumentResumeToken { get; set; }

Add it in the right place, I couldn't comment on a line that didn't change...

Reply:
Once DocumentResumeToken has been added we no longer need the new StartAfter property, so remove it.

We also no longer need the existing ResumeAfter and StartAtOperationTime properties. Not sure if we should remove them since they are a part of the public API. At least deprecate them?

Reply:
Done

Reply:
You still need to remove this new property from the interface.

We're not removing any existing properties, but the only new property we are adding is DocumentResumeToken.